### ISSUE TITLE
ENOTDIR when attempting relay-compiler command

### ISSUE THREAD

**Issue by ryparker** [jelorivera08/MERG-FE#1]
Hey there

I was working through your tutorial when towards the end I was blocked by the following error: 
```~/documents/github/MERG-FE on  master ⌚ 19:51:09
$ yarn relay
yarn run v1.21.1
$ relay-compiler --src ./src --schema ./data/schema.graphql
Error: spawn ENOTDIR
    at ChildProcess.spawn (internal/child_process.js:366:11)
    at Object.spawn (child_process.js:538:9)
    at /usr/local/lib/node_modules/relay-compiler/bin/relay-compiler:1741:31
    at new Promise (<anonymous>)
    at Function.isAvailable (/usr/local/lib/node_modules/relay-compiler/bin/relay-compiler:1735:12)
    at /usr/local/lib/node_modules/relay-compiler/bin/relay-compiler:8706:61
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/usr/local/lib/node_modules/relay-compiler/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/usr/local/lib/node_modules/relay-compiler/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /usr/local/lib/node_modules/relay-compiler/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

This error also occurred when I cloned your repo directly.

**Comment by ryparker** [comment]
After a bunch of debugging I found that the cause was from this line 
`var proc = childProcess.spawn('watchman', ['version']);` in the relay-compiler code. Even the relay source code examples failed here. 

To fix this I installed watchman from https://facebook.github.io/watchman/docs/install.html 
`brew install watchman`

That seemed to fix it. It's strange that nobody else seems to be having this issue. I guess they all have watchman pre-installed?
